reference-architectures/cloud_deploy_flow/CloudRun/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.23 as builder
```

Comment:
Consider refactoring this as a multi-stage build because it seems that you only need `/random-date` to be in the final image.

Reply:
Agreed, I would say let's do this in another PR because I need to learn more about this!